packages/server/main.js

```diff
@@ -6,9 +6,62 @@ import { format, parse } from "json-rpc-protocol";
 // see https://koajs.com/
 const app = new Koa();
 
+var entries = new Map();
+let idCounter = 0;
+
+function generateId() {
+  idCounter += 1;
+}
+
 const METHODS = {
-  subtract([a, b]) {
-    return a - b;
+  createEntry([nameEntry, contentEntry]) {
+    const entry = {
+      id: idCounter,
```

Comment:
`generateId()` should returns a new id, and abstract the implementation details like incrementing `idCounter`.